### PR TITLE
Adding in node backend

### PR DESCRIPTION
**Pull request by fionaphillips**
Resolves [CG-49](https://unosquarebelfast.atlassian.net/secure/RapidBoard.jspa?rapidView=12&projectKey=CG&modal=detail&selectedIssue=CG-49)

nodeFetch

### CODE REVIEW

File: backend/routes/routes.js [hunk 1]
```diff
@@ -0,0 +1,5 @@
+const fetchPDF = require('../services/fetchPDF');
```
Comment (girwin90) [inline backend/routes/routes.js]: I'd put routes.js one level above. The only reason I'd have a routes folder would be if you were having different files such as if it was a ecommerce website with many routes that need seperated product routes, customer routes etc. Hopefully that makes sense.

File: package.json [hunk 2]
```diff
@@ -4,8 +4,10 @@
   "description": "",
   "main": "index.jsx",
   "scripts": {
-    "build": "webpack --config webpack.prod.js",
-    "start:dev": "webpack-dev-server --inline --hot --config webpack.dev.js",
+    "build": " webpack --config webpack.prod.js",
```
Comment (girwin90) [inline package.json:7]: A question more than anything, can you add webpack aliases? I'd assume you could, I haven't tried with node though. It's just be tidier with your routes :)

**Review by girwin90 — CHANGES_REQUESTED** [review]
I know you're only getting going on this, but I thought I'd take a quick look. Just two suggestions so far :)

File: package.json (hunk 2 above)
Comment (fionaphillips) [reply package.json:7]: It's a different process for Node and I've tried a couple of different ways @girwin90 without much success :(. I'll create another ticket for it in the backlog and investigate it separately from this PR.

**Comment by girwin90** [comment]
Just thinking, sorry I skimmed this yesterday. Ideally you'd have this node service in a different repo. That's the way I usually implement this. You'll most likely learn about CORS when trying to get them hooked up which would be a good thing for you to learn.

File: backend/routes.js [hunk 3]
```diff
@@ -0,0 +1,5 @@
+const fetchPDF = require('NodeServices/fetchPDF');
+
+module.exports = app => {
+  app.get('/api/getPDF', (req, res) => fetchPDF(res));
```
Comment (BrownMark) [inline backend/routes.js]: You have constants for this down below right?

**Review by BrownMark — CHANGES_REQUESTED** [review]
Looks good, I have a few other comments but I get the feeling they'll be coming in the next PR :)

File: package.json (hunk 2 above)
Comment (BrownMark) [reply package.json:7]: There are some nice approaches for aliases for Node. I can give you a shout on this Fiona

**Review by BrownMark — APPROVED** [review]

**Pull request merged**